source/javascripts/controllers/_MainController.js.erb

```diff
@@ -99,7 +104,13 @@ import _ from "underscore";
 												"<%= data[:strings][:triggers][:update_deprecated_popup][:details_1] %>",
 												"<%= replaced_string(data[:strings][:triggers][:update_deprecated_popup][:details_2], [data[:routes][:blog][:new_triggers]])%>"
 											])
-												.then(viewModel.save)
+												.then(function() {
+													if (!appService.pipelineConfig.usesRepositoryYml) {
```

Comment:
Remove the negating `!` and switch the bodies of the if-else, so it's easier to read.